librad/src/git/storage.rs

```diff
@@ -493,15 +537,68 @@ impl Storage<WithSigner> {
             return Err(Error::UnsignedMetadata);
         }
 
-        // FIXME: certifier identities must exist, or be supplied
-
         let urn = RadUrn::new(
             meta.root_hash().to_owned(),
             uri::Protocol::Git,
             uri::Path::empty(),
         );
 
+        let self_sig = meta
+            .signatures()
+            .get(&self.signer.public())
+            .ok_or(Error::NotSignedBySelf)?;
+
+        let rad_id = Reference::rad_id(meta.urn().id);
+        let rad_self = Reference::rad_self(meta.urn().id, None);
+        let rad_self_target = match &self_sig.by {
+            Signatory::OwnedKey => rad_id.clone(),
+            Signatory::User(urn) => Reference::rad_id(urn.id.clone()),
+        };
+
+        // Invariants
+        {
+            // Check if we could create `rad/self`, pointing to a valid target
```

Comment:
So I think this comment may be confusing me slightly. The word "could" implies that we have created `rad/self` before this point. But is it actually the case that we're checking if we "can" create a `rad/self` symlink below?

Reply:
Idk, "could" is the subjunctive, no? We _could_ create them _if_ `commit_initial_meta` was successful.

Reply:
Ya, but in this case, it's probably ambiguous. Because "could" is also the past of "can" so I was reading this as "Check if we were able to create rad/self".

I appreciate you changing it :)